packages/code-du-travail-api/package.json

```diff
@@ -52,8 +52,13 @@
     "supertest": "^3.3.0"
   },
   "jest": {
+    "testPathIgnorePatterns": [
+      "node_modules",
+      ".mock.js$"
+    ],
     "collectCoverageFrom": [
       "src/**/*.js",
+      "!src/**/*.mock.js",
```

Comment:
Still required now that mocks are json files ?

Reply:
no no no I update